### ISSUE TITLE
requirements.txt missing from pkg version

### ISSUE THREAD

**Issue by enryH** [KonradUdoHannes/data-steps#2]
Hey Konrad, 

I tried to install directly from pip, but the tar.gz does not contain `requirements.txt` used in `setup.py`

```cmd
pip install data-steps --no-deps
Collecting data-steps
  Using cached data-steps-0.3.0.tar.gz (7.6 kB)
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\user\Anaconda3\envs\vaep\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-k3zihw8j\\data-steps\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-k3zihw8j\\data-steps\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\user\AppData\Local\Temp\pip-pip-egg-info-l_vkekrj'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-k3zihw8j\data-steps\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-k3zihw8j\data-steps\setup.py", line 3, in <module>
        with open("requirements.txt") as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
```

**Comment by KonradUdoHannes** [comment]
Thanks for catching that. I releases a fix with 0.3.1 that fixed it. Maybe the CI should also be modified to not use `pip install .` but to build and then install the package instead, which should catch errors like that.

A quick test installing the package from pypi in a separate repository worked so I'll close the issue.

**Comment by KonradUdoHannes** [comment]
Enhancement to catch these errors already addressed to the most part in #3
